Wikipedia/Code/AppDelegate.m

```diff
@@ -80,6 +116,7 @@ - (void)applicationDidBecomeActive:(UIApplication*)application {
 
 - (void)applicationWillTerminate:(UIApplication*)application {
     // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
+    [self updateDynamicIconShortcutItems];
```

Comment:
would need to check this, but might be redundant if we already call it when going to background